src/autometrics/tracker/test_concurrency.py

```diff
@@ -38,29 +38,23 @@ async def test_concurrency_tracking_prometheus(monkeypatch):
     )
 
 
-# NOTE - Gauges are not supported by the OTEL collector (yet)
-#        https://github.com/open-telemetry/opentelemetry-python/pull/3306
-#
-# @pytest.mark.asyncio
-# async def test_concurrency_tracking_opentelemetry(monkeypatch):
-#     # HACK - We need to set the tracker explicitly here, instead of using `init_tracker`
-#     #        because the library was already initialized with the OpenTelemetry tracker
-#     set_tracker(TrackerType.OPENTELEMETRY)
+@pytest.mark.asyncio
+async def test_concurrency_tracking_opentelemetry(monkeypatch):
```

Comment:
nice!! forgot we could uncomment this test after the otel update

Reply:
yeah i went to update the data there and noticed we can use those now 🎉